packages/v4/src/content/design-guidelines/components/search-input/search-input.md

```diff
@@ -32,9 +32,9 @@ Another way to use search input is as a find function. This would mean that the
 
 3. **Optional navigation:** The navigation feature allows the user to navigate one-by-one through the matched results. It will correspond to the count that is within the badge.
 
-4. **Match highlight:** All matches will be highlighted but we can not guarantee it will contain the gray color, depending on the use case.
+4. **Match highlight:** All matches will be highlighted using `GOLD-50`from the [PatternFly color palette](/guidelines/colors#color-palette).
```

Comment:
should "GOLD-50" be capitalized like that? In the color palette page it's written out in sentence case, ie: gold-50

Reply:
I changed to make consistent.